modules/reactor/src/meshgenerators/PolygonConcentricCircleMeshGeneratorBase.C

```diff
@@ -342,25 +342,55 @@ PolygonConcentricCircleMeshGeneratorBase::PolygonConcentricCircleMeshGeneratorBa
     if (!_ring_block_ids.empty() &&
```

Comment:
add a comment for each of the two statements above

Reply:
will do.